lstchain/calib/camera/flatfield.py

```diff
@@ -78,12 +78,19 @@ def __init__(self, **kwargs):
         self.arrival_times = None  # arrival time per event in sample
         self.sample_masked_pixels = None  # masked pixels per event in sample
 
-        # declare time calibrator if correction file exist
-        if os.path.exists(self.time_calibration_path):
-            self.time_corrector = PulseTimeCorrection(
-                calib_file_path = self.time_calibration_path)
+        # declare time calibrator None of MC events
+        if self.time_calibration_path ==  "None":
```

Comment:
This should not happen! Either it's a string or it's None, but it should not be a string containing `None`.

Why does this happen?

Reply:
Solved:
https://github.com/cta-observatory/cta-lstchain/pull/358/commits/ffc249c8a7723295835e9a2f10b9014f63c9a676